### DISCUSSION TITLE
chi tiết tuần 3

### DISCUSSION THREAD

**Discussion by HuuNghia10A12PhuNhuan** [HoangBao04/10A12-Nhom05#20] [General]
ý tưởng từ nhóm 2 lớp 10A12

**Comment by HuuNghia10A12PhuNhuan** [comment]
![image](https://user-images.githubusercontent.com/92022929/156188288-efecc53a-1e8d-427f-9dff-17a5a74e254f.png)
tối T2, t có coi phần đánh giá của thầy trên Notion. T thấy nhóm 2 đặc biệt được thầy đánh giá là "tiềm năng" nên t đã vào coi thử. Nhóm họ đã yêu cầu thành viên hướng dẫn về cách để định dạng kí tự và văn bản. T nghĩ rằng đây là 1 ý tưởng khá hay nên t sẽ xin phép họ để đc áp dụng cho nhóm mình. Nhưng, t thấy rằng họ vẫn chưa lên bất kỳ ý tưởng nào về 3 câu ca dao, tục ngữ,... Ko biết có phải họ tính để các tuần tới rồi mới làm hay sao? Dù vậy, t vẫn sẽ thảo luận với họ 1 chút nhằm giúp đỡ nhóm ta và nhóm họ hoàn thành bài này thật tốt!

**Reply by HuuNghia10A12PhuNhuan** [reply]
Bonus: nếu thầy có coi đc tin nhắn này, xin thầy hãy đọc phần "chat" trong mục "discussions", tuần trước tụi em thảo luận bên đó ạ.
mục ở phần issues là do 1 vài sai sót nên mới xuất hiện ạ, chứ ko phải là tụi em ko có thảo luận

**Comment by HuuNghia10A12PhuNhuan** [comment]
![image](https://user-images.githubusercontent.com/92022929/156191387-8b27e3e7-da7f-47af-90c0-e2ba382b17ec.png)
có bản quyền rồi nha :)) không cần sợ bị thầy đánh giá là nhái/ăn cắp ý tưởng đâu :))

**Reply by HienCaoPhuNhuan** [reply]
> ![image](https://user-images.githubusercontent.com/92022929/156191387-8b27e3e7-da7f-47af-90c0-e2ba382b17ec.png) có bản quyền rồi nha :)) không cần sợ bị thầy đánh giá là nhái/ăn cắp ý tưởng đâu :))

học bạn là tốt =))))

**Comment by ThuyDung07-A12** [comment]
ê tui chưa hiểu lắm

**Comment by DuyHieu14-10A12** [comment]
cái phần multilevel list là phần kiểu tạo mục lục ấy tui nghĩ khá quan trọng nên mng xem thử cái của t gửi bên project rùi cho ý kiến nha

**Reply by HuuNghia10A12PhuNhuan** [reply]
ok

**Reply by HuuNghia10A12PhuNhuan** [reply]
ôi bạn ơi

**Reply by HuuNghia10A12PhuNhuan** [reply]
sao bạn không tóm tắt lại cho chúng t vậy bạn :))

**Comment by HuuNghia10A12PhuNhuan** [comment]
@ThuyDung07-A12 @HoangBao04 hoàn thành bài đi

**Comment by ThuyDung07-A12** [comment]
hôm nay có chọn bài gì mọi người chừa tui nữa =)) đi học 9h30 mới về mà giờ chưa thấy có bài
thông cảm

**Reply by HuuNghia10A12PhuNhuan** [reply]
bên phần chat tuần trước nha, bên đó chat chính
cái này chỉ để thông báo thui
